functions/src/apis/bikes.ts

```diff
@@ -0,0 +1,161 @@
+import {Response as responseType} from "express";
```

Comment:
I would prefer that in the future we merge one pull-request at a time: https://github.com/jonramm/Bike_Kollective/pull/10